.claude/skills/create-pr/SKILL.md

```
@@ -134,7 +136,289 @@ Hot fix の文脈（`head` が `hotfix/` で始まる、または件名に `Hotf
+            node -e 'process.stdout.write(JSON.stringify({tree:[{path:"README.md",mode:"100644",type:"blob",sha:process.argv[1]}]}))' \
+              "$BLOB" > "$WORK/tree.json"
+            TREE="$(gh api -X POST "repos/$OWNER_REPO/git/trees" --input "$WORK/tree.json" -q .sha)"
+            ROOT_COMMIT="$(gh api -X POST "repos/$OWNER_REPO/git/commits" \
+              -f message='PRスクリーンショット置き場を初期化' -f tree="$TREE" -q .sha)"
+            gh api -X POST "repos/$OWNER_REPO/git/refs" \
+              -f ref="refs/heads/$ASSET_BRANCH" -f sha="$ROOT_COMMIT" >/dev/null
+            ;;
+          *) echo "資材ブランチの確認に失敗（HTTP ステータスを確認してください）" >&2; exit 1 ;;
+        esac
+
+        # 承認後は別プロセスなので、レコードの各行をここでも検証し直す（多層防御）。
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)